src/wp-admin/includes/class-wp-site-health.php

```diff
@@ -2598,8 +2598,11 @@ public function get_autoloaded_options_size() {
 
 		$total_length = 0;
 
-		foreach ( $alloptions as $option_name => $option_value ) {
-			$total_length += strlen( $option_value );
+		foreach ( $alloptions as $option_value ) {
+			if ( is_array( $option_value ) || is_object( $option_value ) ) {
+				$option_value = maybe_serialize( $option_value );
+			}
+			$total_length += strlen( (string) $option_value );
```

Comment:
`maybe_serialize` already handles the `is_array` and `is_object` checks, so this can be simplified further.
```suggestion
			$serialized_value = maybe_serialize( $option_value );
			
			$total_length += strlen( (string) $serialized_value );
```

Reply:
That will end up double-encoding the string, so I don't think that should be done.

Reply:
Oh, I guess not because there is an `is_serialized()` check. I was thinking the logic was the other way around for that double-serialization back-compat quirk.

Reply:
Actually, it will double serialize, which will inflate the total `strlen()` (I just confirmed using XDebug). In this case, what we have here already is probably best.